docs/source/installation/developer-anaconda.rst

```diff
@@ -43,4 +45,63 @@ Quick start
 Detailed instructions
 ---------------------
 
-* Under construction*
+#. Fork the labscript-suite repositories you want to develop using the `GitHub online interface <https://help.github.com/en/github/getting-started-with-github/fork-a-repo>`_. 
+   Below we will include all repositories (except the labscript-suite metapackage).
+
+#. Manually clone the forked repositories using |GitClone|_.
+
+#. Use the `setuptools-conda install-requirements` command to introspect the dependencies from the cloned repositories.
+
+#. Now use `pip` to install the cloned repositories in develop mode without build isolation or installing dependencies via `pip`:
+
+   .. code-block:: console
+
+      C:\Users\wkheisenberg\labscript-suite> pip install --no-build-isolation --no-deps ^
+      -e labscript -e runmanager -e blacs -e lyse -e runviewer -e labscript-devices -e labscript-utils
+
+   .. note::
+        * This will set your forked repository(ies) to be the 'origin' remote.
```

Comment:
I think that the fact that "your forked repository(ies) to be the 'origin' remote." is due to step 2, so maybe this part of the note should be there? Or maybe it looks messy to have two small notes nearby like that?

Reply:
Nice catch! This is what I get for getting to copy-paste happy.

I don't mind having more notes messiness, so I've moved it around.